config/locales/de.requests.yml

```diff
@@ -2,6 +2,43 @@
 
 de:
   requests:
+    templates:
+      headline: "Beispiel-Anfragen"
+      lead_paragraph: "Lorem ipsum dolor sit amet, consectetur adipisicing elit. Laborum fugiat quasi expedita distinctio, suscipit, eaque deleniti similique eveniet in accusantium voluptas blanditiis id modi voluptatibus culpa illum aliquam. Neque, dolore."  
```

Comment:
I don't know whether there is another story for it, but wouldn't it be better if here is a better content than Lorem ipsum?